Jenkinsfile

```diff
@@ -14,6 +14,22 @@ pipeline {
         }
       }
     }
+    stage('reset hardware') {
```

Comment:
need to lock all the resources and do everything at once now

Reply:
I am already locking those with extra, no?

Reply:
`lock(extra: [[resource: "pedal"]], resource: "pandas") {`

Reply:
that's only for the reset stage. the rest of the stages should all be under one lock now